configuration/package_manifests/nfpm.tedge-configuration-plugin.yaml

```diff
@@ -0,0 +1,69 @@
+# yaml-language-server: $schema=https://nfpm.goreleaser.com/static/schema.json
+---
+name: tedge-configuration-plugin
+description: |
+  thin-edge.io configuration management
+arch: "${PKG_ARCH}"
+platform: "linux"
+version: "${GIT_SEMVER}"
+release: "${RELEASE}"
+section: misc
+priority: "optional"
+maintainer: "thin-edge.io team <info@thin-edge.io>"
+vendor: "thin-edge.io"
+homepage: "https://thin-edge.io"
+license: "Apache-2.0"
+
+depends:
+  - tedge
+replaces:
+  - c8y_configuration_plugin
+  - c8y-configuration-plugin
+conflicts:
+  - c8y_configuration_plugin (<= 0.8.1)
+  - c8y-configuration-plugin
+
+deb:
+  fields:
+    Vcs-Browser: ${CI_PROJECT_URL}
+    Vcs-Git: ${CI_PROJECT_URL}
+  compression: xz
+  breaks:
+    - c8y_configuration_plugin (<= 0.8.1)
+    - c8y-configuration-plugin
```

Comment:
I'm confused with this `breaks` clause. The earlier `replaces` and `conflicts` clauses are instructing to replace the old plugin with the new one and here, we're instructing against it. What am I missing?

Reply:
You need both the conflicts/breaks to be defined in order for the tedge-configuration-plugin to automatically replace the c8y-configuration-plugin. This enables a smooth transition from c8y-configuration-plugin to tedge-configuration-plugin.

Reply:
Okay. That automatic replacement is what I was expecting as well. But, the nfpm documentation of the `breaks` key seemed to suggest something totally different:

```
  # Packages which would break if this package would be installed.
  # The installation of this package is blocked if `some-package`
  # is already installed.
  breaks:
    - some-package
```

Which seemed to suggest that attempting to install `tedge-config-plugin` on a system where `c8y-config-plugin` is already installed would be blocked, meaning a smooth upgrade won't be possible. But, I trust you and our system tests more than a possibly misleading doc statement.